src/test/java/uk/gov/hmcts/divorce/notification/CommonContentTest.java

```diff
@@ -50,11 +52,13 @@ void shouldSetCommonTemplateVarsForNotifications() {
 
         final CaseData caseData = caseData();
         caseData.setApplicant2(respondent());
+        when(emailTemplatesConfig.getTemplateVars()).thenReturn(Map.of("divorceCourtEmail", "divorce.court@email.com"));
```

Comment:
Should we also have a test to assert that the court email is correct for dissolution case?

Reply:
done